web/client/src/library/components/documentation/Documentation.tsx

```diff
@@ -277,3 +305,17 @@ Documentation.NotFound = NotFound
 Documentation.Container = Container
 
 export default Documentation
+
+function getValue(value: Primitive): string {
+  const maybeDate = new Date(value as string)
+  const isDate = isString(value) && !isNaN(maybeDate.getTime())
+  const isBoolean = typeof value === 'boolean'
+
+  if (isBoolean && isTrue(value)) return 'True'
+  if (isBoolean && isFalse(value)) return 'False'
+  if (isDate) return toDateFormat(maybeDate)
+  if (isNumber(value)) return value.toString()
+  if (isString(value)) return value
+
+  return ''
```

Comment:
Why not return a string representation of whatever value is instead of an empty string?

Reply:
Also, what if the value is a symbol? Will it reach this line?

Reply:
if payload for `model` details ever change in the future and be an `object` string will look like `[object Object]`

Reply:
In that case, should value not be a primitive? Regardless, I think empty string can be misleading because users might think the value is empty when it actually isn't. Maybe something like "<unknown>" is better?

Reply:
Good point , so in this case I think having `[object Object]` better since it is clearly shows something wrong with representing details and can be caught and fix faster , because `""` can also be interpreted as intentional empty value

Reply:
@mykalmax I was suggesting `"unknown"` above but I think github markup displayed it as "".

Reply:
so with `unknown` it is still possible to see it as intentional value like `References: Unknown` would make sense, but `References: [object Object]` pretty clear is a bug